### PR TITLE
added Order Supplies button

### PR DESCRIPTION
**Pull request by srwendel**
added Order Supplies button that sends the user to amazon or chewy if corresponding apps are present and sends user to web-browser if apps are not present

### CODE REVIEW

File: app/src/main/java/com/pawstime/activities/HomePage.java [hunk 1]
```diff
@@ -106,6 +118,40 @@ public static void clickPetCard(String name, Context context) {
...
+            Uri webpage = Uri.parse(webName);
+            Intent webintent = new Intent(Intent.ACTION_VIEW, webpage);
+
+            PackageManager packageManager = getPackageManager();
+            List<ResolveInfo> activities = packageManager.queryIntentActivities(webintent, 0);
+            boolean isIntentSafe = activities.size() > 0;
+            if (isIntentSafe){
+                startActivity(webintent);
+            }
+            else
```
Comment (sloanlipman) [inline app/src/main/java/com/pawstime/activities/HomePage.java:148]: I'm wondering if you can add a Chooser to the Intent to give the option of opening in a browser if the app isn't installed.

Comment (sloanlipman) [reply app/src/main/java/com/pawstime/activities/HomePage.java:148]: ...or is the block starting on 138 handling that? At the time of writing this comment, I haven't tested. I'm just reading the code.

Comment (srwendel) [reply app/src/main/java/com/pawstime/activities/HomePage.java:148]: The code handles that. If the app is installed it automatically goes there. If the app is not installed, it automatically goes to the browser.

Comment (srwendel) [reply app/src/main/java/com/pawstime/activities/HomePage.java:148]: so yes, 138 is the start of that

**Review by sloanlipman — APPROVED** [review]

**Pull request merged**